triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Server.scala

```diff
@@ -444,7 +447,16 @@ object Server {
           // We got a stop message but haven't completed starting
           // yet. We cannot stop until starting has completed.
           starting(wasStopped = true, req = None)
-        case _ =>
+
+        case TriggerStarting(triggerInstance) =>
+          server.logTriggerStatus(triggerInstance, "s11 starting received in 'starting' state")
+          Behaviors.same
+
+        case TriggerStarted(triggerInstance) =>
+          server.logTriggerStatus(triggerInstance, "s11 running received in 'starting' state")
+          Behaviors.same
```

Comment:
And the hypothesis here was the correct one.

```
  Last failure message: "s11 running received in 'starting' state" did not equal "running". (TriggerServiceTest.scala:202)
```

Reply:
@rohanjr Should our behavior in the `starting` state for these messages equal the behavior in the `running` state?

Reply:
I'm not sure I understand exactly what you mean, but I think the misunderstanding here is due to `TriggerStarted` having a bad name and probably should be called `TriggerRunning`, in which case the log messages are more consistent.

Reply:
Class names would not cause a flaky test, nor would changing them fix it 🙂

Reply:
Oh great catch! Let me summarize my understanding just to make sure I didn’t miss anything:

The issue is that we are starting up the http server in parallel to restarting the triggers which can result in us receiving the `TriggerStarted` message while we are still starting the http server and currently we do not log anything here causing the test to fail.

That definitely looks broken. I see two (non-exclusive) options:
1. Treat the messages the same in starting and in running state. Seems very reasonable to me. We don’t need an http server for this.
2. Only restart triggers after starting the http server. Also seems reasonable to me. I don’t think we need to parallelize this necessarily.

Reply:
@cocreature I'd rather do (1), because it's pretty much done here already, and I don't trust any coordination here. bd8155d4093a711051992581a8b9d835413b5514

Reply:
Makes sense, I was also leaning towards definitely doing 1, we can always do 2 in addition if it makes things easier but no need to do so for now.